db/seeds.rb

```diff
@@ -13,6 +13,21 @@
   puts "#{admin.first_name} #{admin.last_name}, #{admin.email}"
 end
 
+# create applicant
+applicant1 = User.find_or_create_by(email: 'tim@aol.com')
+applicant1.tap do |applicant|
+  applicant.password = 'swordfish'
+  applicant.first_name = 'Tim'
```

Comment:
there are other strategies besides tap.

i though of something about using password... password is not actually a column on our table. devise takes the password and encrypts it, so we do have an encrypted password field on our table. since there is not a password field, we can not use find_or_create_by and pass in password. So we have to separate those things out. That is why the other thing i showed you used find_or_create_by(email: 'some@email.com') and then used create_with(all of the other fields, including password).